packages/design-system/src/styles/components/_SvgIcon.scss

```diff
@@ -90,7 +90,7 @@
 
     path:nth-of-type(1),
     path:nth-of-type(5) {
-      fill: #ffffff;
```

Comment:
A [teeny little super guy](https://www.youtube.com/watch?v=Y8j565lLvJU) of a change - saves a minuscule amount of bytes